app/bt/content.ts

```diff
@@ -30,11 +30,14 @@ export const useBTCopyContent: StrategyCopyContentHook = (t) => {
     moreInfoHowContent: t('exposure_history.bt.how_does_this_work_para'),
     moreInfoWhyContent: t('exposure_history.bt.why_did_i_get_an_en_para'),
     personalPrivacyHeader: t('label.launch_screen2_header_bluetooth'),
+    personalPrivacyIconLabel: t('label.bluetooth_icon'),
     personalPrivacySubheader: t('label.launch_screen2_subheader_bluetooth'),
     notificationDetailsHeader: t('label.launch_screen3_header_bluetooth'),
+    notificationDetailsIconLabel: t('label.heart_icon'),
```

Comment:
If notificationDetailsIconLabel and shareDiagnosisIconLabel are the same for BT and GPS, you don't need to add them to the tracing strategy specific files. These can go right into the en.json and use useTranslation hook.

Reply:
Ah good point, I'll make the update before merging